test/unix/main.ml

```diff
@@ -776,6 +778,43 @@ module Throttle = struct
     ]
 end
 
+(* Tests of {Data.encode} and {Data.encode'} *)
+module Encode = struct
+  module E = I.Private.Data.Entry.Make (Key) (Value)
+  module EDS = I.Private.Data.Entry.Make (KeyDS) (Value)
+
+  let key = Alcotest.(string)
+
+  let keyds = Alcotest.(pair string string)
+
+  let value = Alcotest.(pair string string)
+
+  let encode () =
+    let e =
+      E.v "abcdefghijklmnopqrst" ("AbcdefghijklmnopqrsT", "aBCDEFGHIJKLMNOPQRSt")
```

Comment:
you could use the `random_char` function as in previous tests

Reply:
Yes, will do right now, I just needed them like this to see immediately where were some bugs I had :-)